pipelines/pingcap/tiflow/latest/ghpr_verify.groovy

```diff
@@ -85,13 +84,28 @@ pipeline {
                             }
                         }
                         post {
+                            success {
+                                dir('tiflow') {
+                                    script {
+                                        def testConfigs = [
+                                            dm_unit_test_in_verify_ci: [flags: "unit,dm", coverage_file: "/tmp/dm_test/cov.unit_test.out"],
+                                            unit_test_in_verify_ci: [flags: "unit,cdc", coverage_file: "/tmp/tidb_cdc_test/cov.unit.out"],
+                                            engine_unit_test_in_verify_ci: [flags: "unit,engine", coverage_file: "/tmp/engine_test/cov.unit_test.out"]
```

Comment:
Ref: 
- https://github.com/PingCAP-QE/ci/issues/2171


Let's migrate to use flags to separate test types. components will be identified auto by file path defined in repo's `codecov.yaml` file.

Reply:
Does this PR https://github.com/pingcap/tiflow/pull/9252 need to be cherry-picked to other LTS branches?